### ISSUE TITLE
More info to test monitor flip delays and other bottlenecks that might mess up your experiment timing

### ISSUE THREAD

**Issue by marcobarilari** [cpp-lln-lab/CPP_PTB#37]
From #30 

> @CerenB
> related to these cfg.ifi and cfg.vbl : are they useful if experimenter wants to check the monitor delays? I wanted to test some hardware issues (adapters + which laptop to use). and was thinking flip interval could be a parameter to compare the hardwares.
> 
> @marcobarilari
> This is a good question that tickles a long-standing curiosity. If you want we can dig, I think you can have this info from some PTB's messages and we can add a miniguide to it (e.g. the simplest would be a link in the readme referencing the specific page in the PTB website that, IMHO, is not easy to navigate). I'll open an issue on that.

**Comment by Remi-Gau** [comment]
so the `ifi` is `1 / monitor refresh rate `should be set for a monitor and should not change. I don't think it makes sense to use it for quality control unless you have some high-frequency requirement for your experiment.

**Comment by Remi-Gau** [comment]
the `vbl` is just a timestamp of when a screen was flipped.

**Comment by Remi-Gau** [comment]
for synch QC I would suggest starting with:
- http://psychtoolbox.org/docs/SyncTrouble
- http://psychtoolbox.org/docs/VBLSyncTest

**Comment by CerenB** [comment]
Yes thank you Remi. I had been running those suggestions from SyncTrouble. Mostly I get "your monitor is not good" reports. And seems that only flip rate gives quantitative measure so I can compare across hardwares. I'll further check and note down specifics for other users of CPP_PTB
